packages/client-web/src/modules/user/model/store.ts

```diff
@@ -27,32 +25,24 @@ export const setUserReady = (user: User): SetUserReadyAction => ({
   user,
 });
 
-export const loginUser = (id: string) => (dispatch: AppDispatch) => {
-  dispatch(setUserAuthenticating());
-  return userService.login(id).then((user) => {
-    user.feeds.forEach(({ name, isConnected }) =>
-      dispatch(setFeedConnected(name, isConnected)),
-    );
-
-    dispatch(setUserReady(user));
-  });
-};
```

Comment:
This is in the `user` store. It was dispatching actions for the `feeds` store --> tightly coupled at too low of a level.

New solution is to do this in the `user` view model. View models compose models to implement business logic.